src/App.js

```diff
@@ -36,14 +37,14 @@ function App() {
           </div>
           <Section>
             <h4>Past Events</h4>
-            {pastEvents.length > 0 && pastEvents.map((e) => (
-                <List key={e.id} title={e.title} description={`"on ${e.date} by ${e.speaker}`} />
+            {pastEvents.length > 0 && pastEvents.map((e, i) => (
+              <List key={i} title={e.title} description={`on ${e.date} by ${e.speaker}`} />
```

Comment:
Don't use index as a key.

Reply:
Yeah my bad,
I will update my PR and  remove unique key logic from this PR

And will open a separate issue for Unique key Warning.

@sarabs3 Does this make sense ?

Reply:
Sure. Thank you.
I noticed that API is not returning a unique identifier for the events. I just created a PR for that now.
API code is in gcp-api branch.

Reply:
@sarabs3  - Do we need to have unique identifiers for the events?

Reply:
yes